### PR TITLE
Add: -i flag

### PR DESCRIPTION
**Pull request by stureji**
Modified the program so that it takes flags.

### CODE REVIEW

File: Search.java [hunk 1]
```diff
@@ -9,21 +9,23 @@
  * A small program that will search for strings in a fie.
```
Comment (AzeyZ) [inline Search.java:9]: wrote fie instead of file in comments

File: Search.java [hunk 2]
```diff
@@ -9,21 +9,23 @@
...
+ * java Search <some-string> <file-to-search> [-i]
  *
  * Output:
  * The lines of file-to-search that contains some-string
  */
 class Search {
+    private static boolean flag;
     public static void main(String[] args) {
-        if (args.length != 2) {
+        if (args.length < 2) {
```
Comment (AzeyZ) [inline Search.java:20]: maybe add earlier exit for more than 3 inputs

Comment (stureji) [reply Search.java:20]: Fair point! Thank you so much for this code review!

File: Search.java [hunk 3]
```diff
@@ -17,7 +17,7 @@
 class Search {
     private static boolean flag;
     public static void main(String[] args) {
-        if (args.length < 2) {
+        if (args.length < 2 || args.length > 3) {
```
Comment (AzeyZ) [inline Search.java:20]: Great change

**Review by AzeyZ — APPROVED** [review]

File: Search.java (hunk 1 above)
Comment (stureji) [reply Search.java]: Thank you!

**Pull request merged**